### PR TITLE
refactor: changing email template and adding timezone

### PR DESCRIPTION
**Pull request by yoavweber**
1. changing html file names
2. editing design and adjusting it to html email
3. adding the timezone to the confirmation email

### CODE REVIEW

**Review by MathisBullinger — CHANGES_REQUESTED** [review]
- If you click the cancel buttons in any of the emails, it confirms the event
- The `mailto:EMAIL` link in the in the confirmation email doesn't get substituted for the actual email address
- The `cursor: pointer` on the buttons doesn't get applied in Gmail. Maybe it's an option to just style the link itself like a button instead
- The unprocessed email HTML & CSS shouldn't live as a comment in the final email. Please move them to different files

The message section in the confirmation template is probably not there on purpose…

Screen Shot 2019-12-08 at 16 22 55

File: services/mail.ts [hunk 1]
```diff
@@ -39,6 +39,13 @@ export class Mail {
     return file
   }
 
+  public getTimeZone(date: date) {
+    let getDateObject = new Date(date)
+    let formatDate = Intl.DateTimeFormat(getDateObject)
```
Comment (MathisBullinger) [inline services/mail.ts]: `Intl.DateTimeFormat` expects `string | string[] | undefined` as argument. Passing a `Date` object is a type error and will fail the build. 

Run `npx tsc` to check if the build passes, we should probably add that the CI checks

Comment (fabiofcferreira) [reply services/mail.ts]: We're already running tsc on the CI.

**Comment by MathisBullinger** [comment]
Confirming a meetup results in an error being displayed because the confirmation email couldn't be sent. In the `sendMeetupConfirmation` method you are trying to query `mentorTZ` from the `timeSlots` table. This column doesn't exist in both the dev & production database.

**Review by MathisBullinger — APPROVED** [review]

**Pull request merged**